src/main/kotlin/no/nav/familie/klage/brev/BrevService.kt

```diff
@@ -103,6 +104,7 @@ class BrevService(
     }
 
     private fun lagBrevRequest(
+        behandling: Behandling,
         behandlingId: UUID,
```

Comment:
Kan kanskje fjerne behandlingId nå når vi sender med hele behandlingen (husk metode som kaller denne også)
```suggestion
        behandling: Behandling,
```

Reply:
Fixed: [90a41af](https://github.com/navikt/familie-klage/pull/216/commits/90a41aff2ccb33fc9c075ff25969a01e6a438dd0)